parsons/google/google_bigquery.py

```diff
@@ -44,6 +50,85 @@ def __init__(self, app_creds=None, project=None, location=None):
         # This attribute will be used to hold the client once we have created it.
         self._client = None
 
+    def copy(self, table_obj, dataset_name, table_name, if_exists='fail',
+             tmp_gcs_bucket=None, gcs_client=None):
+        """
+        Copy a :ref:`parsons-table` into Google BigQuery via Google Cloud Storage.
+
+        `Args:`
+            table_obj: obj
+                The Parsons Table to copy into BigQuery.
+            dataset_name: str
+                The dataset name to load the data into.
+            table_name: str
+                The table name to load the data into.
+            if_exists: str
+                If the table already exists, either ``fail``, ``append``, ``drop``
+                or ``truncate`` the table.
+            temp_gcs_bucket: str
+                The name of the Google Cloud Storage bucket to use to stage the data to load
+                into BigQuery. Required if `GCS_TEMP_BUCKET` is not specified.
+            gcs_client: object
+                The GoogleCloudStorage Connector to use for loading data into Google Cloud Storage.
+        """
+        tmp_gcs_bucket = tmp_gcs_bucket or os.environ.get('GCS_TEMP_BUCKET')
```

Comment:
Out of curiosity, is there a reason why you didn't use the `check_env` [util](https://github.com/move-coop/parsons/blob/master/parsons/utilities/check_env.py)?

Reply:
Just forgot about it. I'll swap that out for this.